spec/controllers/stories/bulk_create_controller_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+RSpec.describe Stories::BulkCreateController, type: :controller do
+  let(:project) { create :project }
+  let(:story_1) { attributes_for :story, project: project }
+  let(:story_2) { attributes_for :story, project: project }
+  let(:post_params) do
+    {
+      project_id: project.slug,
+      stories: [story_1, story_2]
+    }
+  end
+
+  context 'when API token matches' do
+    before do
+      ENV['X_API_KEY'] = 'token'
+      headers = { 'HTTP_X_API_TOKEN': ENV['X_API_KEY'] }
+      request.headers.merge headers
+    end
+
+    subject { post :create, params: post_params }
+
+    it 'should return created status' do
+      expect(subject).to have_http_status :created
+    end
```

Comment:
It would be nice to have another scenario making sure that the stories were created, if the service is not called, for example, the spec won't fail.